content/cookbook/wallets/restore-keypair.md

```diff
@@ -22,9 +45,34 @@ const keypair = Keypair.fromSecretKey(
 );
 ```
 
+</Tab>
+
+</Tabs>
+
 ## From base58 String
 
-```typescript filename="restore-keypair-from-base58.ts
+<Tabs groupId="language" items={['web3.js v2 from base58', 'web3.js v1 from base58']}>
+
+<Tab value="web3.js v2 from base58">
+
+```typescript
+import { createKeyPairFromBytes } from "@solana/keys";
+import bs58 from "bs58";
```

Comment:
web3js has a built in base 48 encoder, so you no longer need this package.

please update to use `getBase58Encoder` natively in web3js v2

example: 

```
getBase58Encoder().encode(
    "5MaiiCavjCmn9Hs1o3eznqDEhRwxo7pXiAYez7keQUviUkauRiTMD8DrESdrNjN8zd9mTmVhRvBJeg5vhyvgrAhG",
  ),
```